TicTacToe2/TicTacToe/Board.cs

```diff
@@ -25,19 +25,20 @@ public Board(int size)
             }
         }
 
-        public void PlaceMark(int x, int y, char symbol)
+        public void PlaceMark(Math.Vec2 move, char symbol)
```

Comment:
I recommend adding
```c#
using TicTacToe2.Math;
```
to the top of the file rather than using the Math prefix for the Vec2 type. Vec2 is in a different namespace which is why it can't be used directly without adding the using statement.